crates/tower-cmd/src/run.rs

```diff
@@ -59,9 +60,16 @@ pub fn run_cmd() -> Command {
         .about("Run your code in Tower or locally")
 }
 
+pub async fn do_run(config: Config, args: &ArgMatches, cmd: Option<(&str, &ArgMatches)>) {
```

Comment:
this was a leftover from when I was calling the inner function from the mcp.rs, which I don't do anymore. The idea was that it separates the IO (`eprintln!` and exiting) from the pure part of actually running and reading from a stream. I could probably remove it now, especially since `do_run_inner` isn't an amazing name. However I do like splitting side effectful IO into a wrapper function in general so I didn't feel the urge.